_data/compose-cli/docker_compose_run.yaml

```diff
@@ -83,16 +85,29 @@ options:
   default_value: '[]'
   description: Set environment variables
   deprecated: false
+  hidden: false
   experimental: false
   experimentalcli: false
   kubernetes: false
   swarm: false
-- option: labels
+- option: interactive
+  shorthand: i
+  value_type: bool
+  default_value: "true"
+  description: Keep STDIN open even if not attached.
+  deprecated: false
+  hidden: false
```

Comment:
Looks like for `docker compose exec` we marked this flag `hidden`, but for `docker compose run` we made it visible (perhaps something to look into); or is this dynamically hidden/shown based on the auto-detected?

Reply:
Right! we should apply the same configuration for this tag  as done in the `exec` command